src/app/login/login.component.ts

```
@@ -116,6 +122,8 @@
               this.authService.login(this.userName, encryptPassword, true).subscribe((userLoggedIn) => {
                 if (userLoggedIn.statusCode == '200') {
                 if (userLoggedIn.data.previlegeObj != null && userLoggedIn.data.previlegeObj != undefined && userLoggedIn.data.previlegeObj[0]) {
+                  this.cookieService.set('Jwttoken', userLoggedIn.data.Jwttoken);
+                  delete userLoggedIn.data.Jwttoken;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)